src/components/keyboard/keyboard.js

```diff
@@ -136,6 +136,13 @@ class KeyboardControl extends BaseComponent {
         return true;
       }
     },
+    [KeyMap.P]: () => {
+      if (!this.player.isInPictureInPicture()) {
```

Comment:
if you are in focus on screen can we toggle back to out of PiP? if so add the `else` clause for it

Reply:
@OrenMe I can see that that the option to exit PiP is available and working via script (console), however, the limitation is on the keybinding, it seems that when on PiP mode, the player bindings are getting overridden.

Following the above, do you still want me to implement the `else` clause?

Reply:
ok, but if you focus back on page and click it will toggle? if so please add it

Reply:
Added